### PR TITLE
Add section on configuring unprivileged singularity

### PR DESCRIPTION
**Pull request by DrDaveD**

### CODE REVIEW

**Comment by brianhlin** [comment]
@matyasselmeci these instructions should link to our documentation about setting arbitrary environment variables in `osg-configure`. Where would that be?

**Comment by matyasselmeci** [comment]
40-localsettings.ini; https://opensciencegrid.org/docs/other/configuration-with-osg-configure/#local-settings

File: docs/worker-node/install-singularity.md [hunk 1]
```diff
@@ -142,6 +142,24 @@ steps:
...
+
+### Configuring Unprivileged Singularity ###
+
+When unprivileged singularity is enabled and VOs run singularity from
+CVMFS, the singularity configuration file also comes from CVMFS so local
+sites have no control over changing the configuration.  However, the
+most common local configuration change to the singularity RPM is to add
+additional local "bind path" options to map extra local file paths into
+containers.  This can instead be accomplished by setting the
+`SINGULARITY_BINDPATH` variable in the environment of jobs.  This is a
```
Comment (brianhlin) [inline docs/worker-node/install-singularity.md:154]:
```suggestion
`SINGULARITY_BINDPATH` variable in the environment of jobs through
[configuration](/other/configuration-with-osg-configure/#local-settings)
on your compute element.  This is a
```

Comment (DrDaveD) [reply docs/worker-node/install-singularity.md]: I'm going to change that to say that's one way to do it.

**Pull request merged**